packages/x-charts/src/RadarChart/RadarChart.tsx

```diff
@@ -61,8 +60,9 @@ const RadarChart = React.forwardRef(function RadarChart(
         <ChartsSurface {...chartsSurfaceProps} ref={ref}>
           <RadarGrid {...radarGrid} />
           <RadarMetricLabels />
-          <RadarSeriesPlot />
-          {highlight === 'axis' && <RadarAxisHighlight {...radarAxisHighlight} />}
+          <RadarSeriesArea />
+          {highlight === 'axis' && <RadarAxisHighlight />}
+          <RadarSeriesMarks />
```

Comment:
I'm wondering if I should remove this `RadarSeriesPlot`

The reason to have to component is to allow to switch between the two staking strategies:

1. *per series* strategy (the one of RadarSeriesPlot)
	area1, marks1, area2, marks2, area3, marks3
2.  *per visual* strategy (the one of RadarSeriesArea + RadarSeriesMarks)
	area1, area2, area3, marks1, marks2, area3, marks3

Reply:
What would the benefit of removing `RadarSeriesPlot` be?

> per series strategy (the one of RadarSeriesPlot)

Shouldn't we always use this one to ensure depth is correctly represented? I.e., that the first series' mark color is affected by the background color of the second series area?

Reply:
For the axis highlight, I needed to split it such that the doted axis line and the highlighted mark shadow areas on top of all area, but below the marks

The advantage of removing it would be to avoid code duplicate. But I might found a better solution to share props generation between the two.

Reply:
Seem you removed it already? It doesn't look like a `per series` strategy is desirable, so the current behaviour should be good.

Reply:
Not really. From a rendering point of view, both make sense.

My concern was the fact porps like `r={3}` was passed manually in the two components. The issue I've face is that I updated on component and not the other, leading to some inconsistencies.

I last commit, I created `getCircleProps` and `getPathProps` to have a single place to transform highlight stats to props